src/components/DigitalCredentials/IssueCredentials.vue

```diff
@@ -121,7 +121,7 @@ export default class IssueCredentials extends Vue {
     }
   }
   .alert-text {
-    color: $BCgovGold9;
+    color: #F8661A;
```

Comment:
Should use named colour (CSS variable).

Also please update app version.

Reply:
Make sense but please add it anyway.

Is it different from existing red?

Reply:
It is unique, yes 😬 
It's a very special message!